### ISSUE TITLE
Loading a specific song only if the player is on story mode

### ISSUE THREAD

**Issue by charaster** [ShadowMario/FNF-PsychEngine#13728]
### What is your question?

is it possible to load a specific song but only when the player is on story mode and before you say "just add it to the week.json" no i want to stay hidden till the player actually plays the week and finishes the song before it. i took a look at some old lua code from 0.6.3 but as i thought that dint work but the code i stiched together looks like this 

"if isStoryMode then
    function onEndSong()
        loadSong('finale','goofy')
    end
end" 

tl;dr: is it possible to load a song without having it appear in the story mode menu?
